### PR TITLE
Draft  Alerts `README.md`

### PR DESCRIPTION
**Pull request by emily-curtis**
This pull request contains the `README.md` for the `alerts/` folder in the codebase. 
The file outlines:
1. A general overview of the alerts folder and what it does
2. The scripts contained within the folder and their functionality
3. Details on how to set up and run the project

### CODE REVIEW

File: alerts/README.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+    ├── queries.py
+    ├── utilities.py
+    ├── requirements.txt
+```
+
+### **Scripts**
+1. **`alerts.py`**
+   - The script monitors changes in top artists, genres, and specific subscribed genres on Bandcamp, generating alerts for significant shifts, such as changes in top rankings or spikes in sales, and sends notifications to subscribers via email.
+   - It sends personalized email alerts for various scenarios, including a change in the top artist or genre, and sales growth in user-subscribed genres, using SMTP for reliable email delivery.
+   - By utilizing SQL queries through a database connection, the script fetches real-time and historical data to calculate metrics like sales deltas and identify changes, ensuring alerts are based on accurate and up-to-date information.
```
Comment (ebradley12) [inline alerts/README.md:34]: Can we change to English spelling 'utilising'

File: alerts/README.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+    ├── lambda_handler.py
+    ├── queries.py
+    ├── utilities.py
+    ├── requirements.txt
+```
+
+### **Scripts**
+1. **`alerts.py`**
+   - The script monitors changes in top artists, genres, and specific subscribed genres on Bandcamp, generating alerts for significant shifts, such as changes in top rankings or spikes in sales, and sends notifications to subscribers via email.
+   - It sends personalized email alerts for various scenarios, including a change in the top artist or genre, and sales growth in user-subscribed genres, using SMTP for reliable email delivery.
```
Comment (ebradley12) [inline alerts/README.md:33]: personalised*

**Review by ebradley12 — APPROVED** [review]
Fab!

File: alerts/README.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+## **Setting Up**
+
+### **Prerequisites**
+- Python 3.8 or higher.
+- Access to the database (RDS or local setup) with the required schema.
+- AWS credentials configured for:
+    - Sending alert emails via AWS SES.
+
+### **Installing Dependencies**
+#### **For Alerts Folder Only**
```
Comment (GabrielNsiah) [inline alerts/README.md:63]: Would it be better to say "For the Alerts folder Only?"

**Review by bentrzcinski — APPROVED** [review]

**Pull request merged**